src/cil.ml

```diff
@@ -4616,7 +4936,7 @@ class defaultCilPrinterClass : cilPrinter = object (self)
 
 end (* class defaultCilPrinterClass *)
 
-let defaultCilPrinter = new defaultCilPrinterClass
+let defaultCilPrinter = new cilAstDumpClass
```

Comment:
Let's not change the default one to this, we should leave it as it as.

Reply:
Just wanted to ask.
What's the difference to `defaultCilPrinterClass`? Does it just pretty print as valid C?

Reply:
Yes, it is e.g. the one we use for `--enable justcil` in Goblint.